lib/linguist/languages.yml

```diff
@@ -860,6 +860,16 @@ C2hs Haskell:
   codemirror_mode: haskell
   codemirror_mime_type: text/x-haskell
   language_id: 45
+C3:
+  type: programming
+  color: "#969696"
+  extensions:
+  - ".c3"
+  - ".c3i"
+  - ".c3l"
```

Comment:
All extensions being added need to meet our usage requirements and must have samples. If any one extension has lower user than others, it'll hold back the entire PR until usage is sufficient.